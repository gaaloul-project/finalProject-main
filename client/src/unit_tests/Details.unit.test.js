import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Section1 from '../src/Section1.jsx.js';
import Section2 from '../src/Section2.jsx.js';
import Section3 from '../src/Section3.jsx.js';
import Section4 from '../src/Section4.jsx.js';

it('shallow renders the_space correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Section1 ltr={'TheSpace'} />);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});

it('shallow renders Guess Access correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Section2 ltr={'GuestAccess'} />);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});

it('shallow renders Interaction With Guests correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Section3 ltr={'InteractionWithGuests'} />);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});

it('shallow renders Other Things to Note correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Section4 ltr={'OtherThingsToNote'} />);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});
