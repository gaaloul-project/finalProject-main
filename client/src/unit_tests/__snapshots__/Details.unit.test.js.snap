// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow renders Guess Access correctly 1`] = `
<div
  dir="ltr"
>
  <styled.p>
    <styled.span>
      <styled.span>
        GuestAccess
      </styled.span>
    </styled.span>
  </styled.p>
</div>
`;

exports[`shallow renders Interaction With Guests correctly 1`] = `
<div
  dir="ltr"
>
  <styled.p>
    <styled.span>
      <styled.span>
        InteractionWithGuests
      </styled.span>
    </styled.span>
  </styled.p>
</div>
`;

exports[`shallow renders Other Things to Note correctly 1`] = `
<div
  dir="ltr"
>
  <styled.p>
    <styled.span>
      <styled.span>
        OtherThingsToNote
      </styled.span>
    </styled.span>
  </styled.p>
</div>
`;

exports[`shallow renders the_space correctly 1`] = `
<div
  dir="ltr"
>
  <styled.p>
    <styled.span>
      <styled.span>
        TheSpace
      </styled.span>
    </styled.span>
  </styled.p>
</div>
`;
